package.json

```diff
@@ -11,6 +11,7 @@
     "braces": ">=2.3.1",
     "canvas": "^2.6.0",
     "crypto-js": "^3.1.9-1",
+    "embedme": "^1.18.1",
```

Comment:
@bassemmagdy It's not necessary to add ``embedme`` as a dependency of the package since we are running it globally with ``npx embedme README.md``